src/lib/keccak.unit-test.ts

```diff
@@ -0,0 +1,105 @@
+import { Field } from './field.js';
+import { Provable } from './provable.js';
+import { ZkProgram } from './proof_system.js';
+import { constraintSystem, print } from './testing/constraint-system.js';
+import {
+  ROUND_CONSTANTS,
+  theta,
+  piRho,
+  chi,
+  iota,
+  round,
+  blockTransformation,
+} from './keccak.js';
+
+const KECCAK_TEST_STATE = [
```

Comment:
Because this is still just a portion of the hash function as it does not contain padding, sponges, etc, this test is not sufficient for the final implementation. It will be clearer in the future if you can include full hash tests which I guess is already planned. In any case, this looks ok for now.